src/core/src/service.rs

```diff
@@ -256,6 +256,11 @@ impl Poet2Service {
 
          serial_cert.clone()
     }
+
+    pub fn get_next_wait_time( &mut self ) -> u64 {
+        debug!("Getting new wait time for next block.");
+        get_next_wait_time( 5.5_f64 )
```

Comment:
please choose a better name to indicate this is the method called from enclave_sim.rs